pkgs/ok_http/lib/src/ok_http_client.dart

```diff
@@ -44,13 +44,40 @@ import 'jni/bindings.dart' as bindings;
 /// ```
 class OkHttpClient extends BaseClient {
   late bindings.OkHttpClient _client;
+  bool _isClosed = false;
 
   OkHttpClient() {
     _client = bindings.OkHttpClient.new1();
   }
 
+  @override
+  void close() {
+    if (!_isClosed) {
```

Comment:
Could you include the URL where the methods to call were listed?

Reply:
> Could you include the URL where the methods to call were listed?

Done! ✅